sdk/python/feast/batch_feature_view.py

```diff
@@ -0,0 +1,61 @@
+from datetime import timedelta
+from typing import Dict, List, Optional, Union
+
+from google.protobuf.duration_pb2 import Duration
+
+from feast import Entity, Feature, Field
+from feast.data_source import DataSource
+from feast.feature_view import FeatureView
+from feast.protos.feast.core.DataSource_pb2 import DataSource as DataSourceProto
+
+SUPPORTED_BATCH_SOURCES = {
+    "BigQuerySource",
+    "FileSource",
+    "RedshiftSource",
+    "SnowflakeSource",
+    "SparkSource",
+    "TrinoSource",
+}
+
+
+class BatchFeatureView(FeatureView):
+    def __init__(
+        self,
+        *,
+        name: Optional[str] = None,
+        entities: Optional[Union[List[Entity], List[str]]] = None,
+        ttl: Optional[Union[Duration, timedelta]] = None,
+        features: Optional[List[Feature]] = None,
```

Comment:
is BFV meant to be backwards compatible with FV? if not, we should just eliminate this now and switch to `schema`

and make all the changes to other parameters, e.g. `ttl` should not support a `Duration` object

Reply:
good catch. The new classes don't need to support the old API.